docker-compose-test.yml

```diff
@@ -7,7 +7,6 @@ services:
     environment:
       - DEBUG=False
     volumes:
-      - ./app:/app
```

Comment:
I realized we don't need to volume map this in during the test run since it always copies the freshest version at build time.